src/frontend/screens/ObservationFields/index.tsx

```diff
@@ -37,10 +38,17 @@ export const ObservationFields = ({
   navigation,
   route,
 }: NativeRootNavigationProps<'ObservationFields'>) => {
+  const {projectId} = useActiveProject();
+  const languageTag = useAppLanguageTag();
+
+  const {data: fields} = useManyDocs({
+    projectId,
+    docType: 'field',
+    lang: languageTag,
+  });
```

Comment:
great example of how to use these hooks when there's no custom/shared logic that's needed across the app. there are definitely more places in the app where we can get rid of wrapper hooks in favor of something like this, but probably best to do as a follow-up.